web/covid_web/country_dashboard.py

```diff
@@ -178,42 +170,27 @@ def _create_buttons(
         return buttons
 
     def _create_single_country_callbacks(self, app: dash.Dash) -> None:
-        @app.callback(
-            [
-                Output("country-short-name", component_property="options"),
-                Output("graph-title", component_property="children"),
-            ],
-            [Input("prediction-event", "value")],
-        )
-        def update_event(prediction_event_name):
-            options = [
-                dict(label=graph.long_name, value=graph.short_name)
-                for graph in self.graphs_by_event[prediction_event_name]
-            ]
-            prediction_date = self.prediction_event_by_name[prediction_event_name].prediction_date
-            return options, f"{prediction_date.strftime('%B %d')} predictions"
+        # TODO: Predictions will be generated statically in the future.
+        graphs_by_country = {
+            country_short_name: CountryGraph(
+                self.report_by_short_name[country_short_name],
+                get_fitted_predictions(
+                    self.report_by_short_name[country_short_name],
+                    dates=self.report_by_short_name[country_short_name].dates[-14:],
+                ),
+            )
+            for country_short_name in prediction_db.get_countries()
+        }
 
         @app.callback(
             Output("country-graph", component_property="figure"),
-            [
-                Input("prediction-event", "value"),
```

Comment:
So we allow sliding through the two BK predictions now? I am a bit confused now how `SingleCountry` page functionality is changed. Also do we now mix BK and Daily predictions on the daily predictions page, or BK is always visible?

Reply:
Single country prediction is for automatic predictions only. I've mentioned this on Slack - to keep the number of web pages and dashboard manageable, we have to drop functionality from time to time.

BK predictions are still visible on the `/covid19/all` page, which is the most popular one anyway :smile: